booking/app/marshallers.py

```diff
@@ -0,0 +1,20 @@
+from flask_restful import fields
+
+
+event_fields = {
+    'id': fields.String,
+    'title': fields.String,
+    'ticket_price': fields.Integer,
+    'sale_start': fields.String,
+    'sale_end': fields.String,
+    'max_tickets_in_order': fields.Integer,
+    'order_expiration_period': fields.String,
+}
+
+order_fields = {
+    'id': fields.String,
+    'event': fields.String(attribute=lambda order: order.event.id),
+    'ticket_holder_name': fields.String,
+    'billing_address': fields.String,
+    'order_open_time': fields.String,
```

Comment:
This field shouldn't be filled by user, probably this should be Datetime, because then we use this info in calculating activate property for TickerOrder class.